packages/patternfly-4/react-core/src/components/EmptyState/EmptyStateIcon.tsx

```diff
@@ -11,8 +11,11 @@ export enum IconSize {
 }
 
 export interface IconProps extends Omit<React.HTMLProps<SVGElement>, 'size'> {
+  /** @deprecated */
```

Comment:
can you remove the `@` since we do not supported `@deprecated` in our documentation.